docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,495 @@
+# Danceable Lyrics
+In this guide, we use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that connects directly to a set of data files and determines the most "danceable" songs that contain a set of lyrics. We use [Polars](https://www.pola.rs/) for fast data processing.
+
+## Table of Contents
+* [Download Input Data](#download-input-data)
+* [Basic Plugin Setup](#basic-plugin-setup)
+* [Write a Plugin](#write-a-plugin)
+    * [Dependencies](#1-dependencies)
+    * [Imports](#2-imports)
+    * [Initialization](#3-initialization)
+    * [Data Processing](#4-data-processing)
+    * [Putting It All Together](#5-putting-it-all-together)
+* [Packaging into a YXI](#package-into-a-yxi)
+* [Run the Test Client](#run-the-test-client)
+* [Install in Designer](#install-and-run-in-designer)
+    * [Method 1](#method-1)
+    * [Method 2](#method-2)
+* [Run in Designer](#run-in-designer)
+
+## Download Input Data
+Download and extract the required dataset files. You will reference them later in this guide. We provide 2 datasets—the full dataset and a smaller, truncated version.
+
```

Comment:
A preview (couple rows) of what the raw data looks like would be great!!! We want to show off the complexity of the dataset. (JSON... )